ml-agents/mlagents/trainers/tests/test_simple_rl.py

```diff
@@ -402,7 +402,7 @@ def test_simple_asymm_ghost_fails(use_discrete):
     processed_rewards = [
         default_reward_processor(rewards) for rewards in env.final_rewards.values()
     ]
-    success_threshold = 0.99
```

Comment:
do we want to document why this changes over time?  Maybe a comment about why it was changed along with the date.